test.py:22: error: Rule 'sec_rule' is already present in the grammar
Done
